tests/exporters/test_tasks_manager.py

```diff
@@ -0,0 +1,30 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+from optimum.exporters import TasksManager
+
```

Comment:
why not include it to [tests/exporters/common/test_tasks_manager.py](https://github.com/huggingface/optimum/blob/main/tests/exporters/common/test_tasks_manager.py) ?

Reply:
Did not know this one existed